src/plugins/intel_cpu/src/nodes/pooling.cpp

```diff
@@ -146,6 +146,14 @@ dnnl::pooling_forward::primitive_desc createDescriptorHelper(const dnnl::engine&
 }  // namespace
 
 bool Pooling::isSupportedOperation(const std::shared_ptr<const ov::Node>& op, std::string& errorMessage) noexcept {
+#if defined(OV_CPU_WITH_ACL)
+    if (auto maxPoolOpBase = ov::as_type_ptr<const ov::op::util::MaxPoolBase>(op)) {
+        if (maxPoolOpBase->get_kernel() != ov::Shape(2,2)) {
+            errorMessage = "Pooling indices returning source tensor coordinates is only supported for pool size 2x2";
+            return false;
+        }
+    }
+#endif
```

Comment:
It should be inside try-catch block below, after common checks

Reply:
Fixed